### PR TITLE
Setting up compliance checks

### PR DESCRIPTION
**Pull request by BSick7**
This PR adds compliance checks and badges.

### CODE REVIEW

File: fargate.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 resource "aws_ecs_cluster" "this" {
```
[the remark this answers is not in the document]
Comment (ssickles) [reply fargate.tf:1]: This is cool!

File: README.md [hunk 2]
```diff
@@ -2,15 +2,41 @@
...
+[![PCI-DSS V3.2](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/pci)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=PCI-DSS+V3.2)
+[![NIST-800-53](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/nist)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=NIST-800-53)
+[![ISO27001](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/iso)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=ISO27001)
+[![SOC2](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/soc2)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=SOC2)
+[![HIPAA](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/hipaa)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=HIPAA)
+
 ## Inputs
 
+- `versioning: bool`
```
Comment (ssickles) [inline README.md]: These two inputs look like they are for S3. Did these accidentally get copied in?

**Review by ssickles — CHANGES_REQUESTED** [review]
Looks like we still have a BridgeCrew issue to resolve. Also, check out my one comment.

File: README.md (hunk 2 above)
Comment (BSick7) [reply README.md]: Good catch. Resolved.

**Review by ssickles — APPROVED** [review]
LGTM!

**Pull request merged**